ddocs/builds/validate_doc_update.js

```diff
@@ -1,6 +1,10 @@
-function(newDoc, oldDoc, userCtx, secObj) {
+function(newDoc, oldDoc, userCtx) {
   var version = require('views/lib/version').fn(newDoc._id);
 
+  if (!userCtx || userCtx.roles.indexOf('builds-admin') === -1) {
```

Comment:
This is the main bit really..

Reply:
Hardcoding the role is fine by me. 
We could also allow all roles assigned as db admins + the `_admin` role (like in https://github.com/medic/medic/blob/master/ddocs/medic/validate_doc_update.js#L30)